### PR TITLE
Tidy up validate-modules:doc-elements-mismatch

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
Tidy up on sanity tests' ignore files, removing lines with validation-modules for these modules.

The focus was to pass the tests so no code was changed other than adjusting argument_spec for missing types or inconsistencies with the docs. The priority has been not to change any functionality, but to ensure that code and docs are synchronized.

Some lines were added back to the files, but a total of 26 lines were removed (among 2.9, 2.10 and 2.11).

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
plugins/modules/web_infrastructure/sophos_utm/utm_proxy_exception.py
plugins/modules/source_control/github/github_webhook.py
plugins/modules/remote_management/redfish/redfish_config.py
plugins/modules/net_tools/nios/nios_nsgroup.py
plugins/modules/net_tools/nios/nios_network.py
plugins/modules/net_tools/nios/nios_member.py
plugins/modules/net_tools/nios/nios_host_record.py
plugins/modules/net_tools/nios/nios_fixed_address.py
plugins/modules/identity/keycloak/keycloak_clienttemplate.py
plugins/modules/identity/keycloak/keycloak_client.py
plugins/modules/identity/ipa/ipa_hbacrule.py
plugins/modules/cloud/xenserver/xenserver_guest.py

### CODE REVIEW

File: changelogs/fragments/1399-fixed-wrong-elements-type.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- utm_proxy_exception - four parameters had elements types set as 'string' (invalid), changed to 'str'. (https://github.com/ansible-collections/community.general/pull/1399)
```
Comment (felixfontein) [inline changelogs/fragments/1399-fixed-wrong-elements-type.yaml:2]:
```suggestion
- utm_proxy_exception - four parameters had elements types set as 'string' (invalid), changed to 'str' (https://github.com/ansible-collections/community.general/pull/1399).
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
Some tests are still failing: https://app.shippable.com/github/ansible-collections/community.general/runs/5595/2/tests

**Comment by russoz** [comment]
Yup, on it.

**Review by felixfontein — APPROVED** [review]

**Comment by russoz** [comment]
Johnny on the spot! Thanks @felixfontein

**Comment by felixfontein** [comment]
@russoz thanks!

**Pull request merged**